CONTRIBUTING.md

```diff
@@ -12,6 +12,29 @@ We have provided an [issue template](.github/bug.md) what will help guide you th
 If you are unsure if something is a bug, the `#ember-data` channel on [Discord](https://discord.gg/zT3asNS) is
 a great place to ask for help!
 
+#### Testing ember data source directly
```

Comment:
this is for depending on monolithic ember-data but not individual packages. It does solve the transitive dependency problem.

Reply:
updates before merging
- generalize the language for individual packages
- yarn + npm do not mix so we need to caveat and describe this better (see ember.js discussion)